staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanagertest/testfieldmanager.go

```diff
@@ -125,22 +126,22 @@ func NewTestFieldManager(typeConverter fieldmanager.TypeConverter, gvk schema.Gr
 	live := &unstructured.Unstructured{}
 	live.SetKind(gvk.Kind)
 	live.SetAPIVersion(gvk.GroupVersion().String())
-	f = fieldmanager.NewLastAppliedUpdater(
-		fieldmanager.NewLastAppliedManager(
-			fieldmanager.NewProbabilisticSkipNonAppliedManager(
-				fieldmanager.NewBuildManagerInfoManager(
-					fieldmanager.NewManagedFieldsUpdater(
-						fieldmanager.NewStripMetaManager(f),
+	f = internal.NewLastAppliedUpdater(
```

Comment:
Document why this construction is different than the one in the NewDefaultFieldManager function?